bigchaindb/core.py

```diff
@@ -365,15 +285,7 @@ def validate_transaction(self, transaction):
                     raise exceptions.DoubleSpend('input `{}` was already spent'.format(
                         transaction['transaction']['input']))
 
-        # Check hash of the transaction
```

Comment:
Usually I try to not "over plan", but I'm quite sure we will need this also in the client code to check the validity of the data coming from the Federation.